src/main/java/core/basesyntax/LambdaCalc.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+public class LambdaCalc {
+    public static double calculate(int firstNumber, int secondNumber, char operation) {
```

Comment:
```suggestion
    public double calculate(int firstNumber, int secondNumber, char operation) {
```
let's avoid creating static methods